gobblin-metrics/build.gradle

```diff
@@ -85,7 +85,7 @@ artifacts {
 
 test {
     useTestNG () {
-      excludeGroups 'ignore'
+      excludeGroups 'ignore', 'performance'
```

Comment:
Do you mean a different project? I wanted to avoid having a different project just for performance of metrics. Also, the tests are fairly slow, so I didn't want them to run with unit tests.
I can add a method to gradle (say `gradle performance`) to run the performance tests, would this work?